Dockerfile

```diff
@@ -27,6 +27,7 @@ FROM debian:buster-20191118-slim as azure-cli
 ARG AZURE_CLI_VERSION
 ARG PYTHON_MAJOR_VERSION
 RUN apt-get update
+RUN apt-get install -y python3-distutils
```

Comment:
@ybelMekk `python3-distutils` do the trick but it need to be moved out of this build step to the final image like so :

```docker
RUN apt-get update \
  && apt-get install -y --no-install-recommends \
    ca-certificates=20190110 \
    git=1:2.20.1-2+deb10u3 \
    python3=${PYTHON_MAJOR_VERSION}.3-1 \
   python3-distutils=3.7.3-1 \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/* \
  && update-alternatives --install /usr/bin/python python /usr/bin/python${PYTHON_MAJOR_VERSION} 1
```

Reply:
Okei, fixing it